deepchem/models/torch_models/ferminet.py

```diff
@@ -118,12 +122,14 @@ def forward(self, input: np.ndarray) -> torch.Tensor:
             contains the wavefunction - 'psi' value. It is in the shape (batch_size), where each row corresponds to the solution of one of the batches
         """
         # creating one and two electron features
+        # torch.autograd.set_detect_anomaly(True)
+        eps = torch.tensor(1e-36)
```

Comment:
adding epsilon/eps so that the derivative of norm of x - 1/sqrt(x) will be defined even when x is 0.

Reply:
Make this a comment as well